### PR TITLE
renames

### PR DESCRIPTION
**Pull request by rperez89**
Renaming cache struct and functions like 'popX'

### CODE REVIEW

File: contracts/DandelionOrg.sol [hunk 1]
```diff
@@ -406,81 +406,81 @@ contract DandelionOrg is BaseTemplate {
         _createTimeLockPermissions(_acl, timeLock, dandelionVoting, dandelionVoting, tokenBalanceOracle);
     }
 
-    // CACHE FNS
-    function _cacheToken(MiniMeToken _token) internal {
-        Cache storage c = cache[msg.sender];
+    // SAVE FNS
+    function _saveToken(MiniMeToken _token) internal {
+        DeployedContracts storage contracts = deployedContracts[msg.sender];
```
Comment (willjgriff) [inline contracts/DandelionOrg.sol]: Nitpicky but I would call this something like `senderDeployedContracts` to make it clear what this is when reading it mid-function.

File: contracts/DandelionOrg.sol [hunk 2]
```diff
@@ -14,8 +14,8 @@ import "openzeppelin-solidity/contracts/token/ERC20/ERC20Detailed.sol";
 contract DandelionOrg is BaseTemplate {
     string constant private ERROR_EMPTY_HOLDERS = "DANDELION_EMPTY_HOLDERS";
     string constant private ERROR_BAD_HOLDERS_STAKES_LEN = "DANDELION_BAD_HOLDERS_STAKES_LEN";
-    string constant private ERROR_MISSING_CACHE = "DANDELION_MISSING_CACHE";
-    string constant private ERROR_MISSING_TOKEN_CACHE = "DANDELION_MISSING_TOKEN_CACHE";
+    string constant private ERROR_MISSING_CONTRACT = "DANDELION_MISSING_CONTRACT";
```
Comment (willjgriff) [inline contracts/DandelionOrg.sol]: Could be `MISSING_DAO_CONTRACT`

File: contracts/DandelionOrg.sol [hunk 3]
```diff
@@ -406,81 +406,81 @@ contract DandelionOrg is BaseTemplate {
...
-    function _ensureBaseAppsCache() internal {
-        Cache storage c = cache[msg.sender];
-        require(c.finance != address(0), ERROR_MISSING_CACHE);
-        require(c.tokenManager != address(0), ERROR_MISSING_CACHE);
-        require(c.agentOrVault != address(0), ERROR_MISSING_CACHE);
+    function _ensureBaseAppsDeployed() internal {
+        DeployedContracts storage contracts = deployedContracts[msg.sender];
+        require(contracts.finance != address(0), ERROR_MISSING_CONTRACT);
+        require(contracts.tokenManager != address(0), ERROR_MISSING_CONTRACT);
+        require(contracts.agentOrVault != address(0), ERROR_MISSING_CONTRACT);
```
Comment (willjgriff) [inline contracts/DandelionOrg.sol]: I think these would be better as different errors, one for each. Up to you though.

**Review by willjgriff — APPROVED** [review]

File: contracts/DandelionOrg.sol (hunk 1 above)
Comment (rperez89) [reply contracts/DandelionOrg.sol]: what you are suggesting here is to change the 'contracts' variable name right?

Comment (willjgriff) [reply contracts/DandelionOrg.sol]: Yeah

**Review by fabriziovigevani — APPROVED** [review]
LGTM!

I think we only have to change the `contracts` renaming suggested by Will and we can merge it

**Pull request merged**